### PR TITLE
Fix webpack roots

### PR DESCRIPTION
**Pull request by jasonLaster**
### Summary of Changes

We had an issue where webpack based urls had their paths changed in the tree from `webpack:///src/file.js` to `Webpack/src/file.js` changing the source url meant that the root no longer lined up with the source urls. This patch fixes that!

---

### Before
screen shot 2018-04-05 at 9 38 16 am

### After
screen shot 2018-04-05 at 9 19 00 am

### CODE REVIEW

File: src/selectors/getRelativeSources.js [hunk 1]
```diff
@@ -31,10 +31,9 @@ function formatSource(source: Source, root): RelativeSource {
 export function getRelativeSources(state: State): RelativeSource[] {
   const sources = getSources(state);
   const root = getProjectDirectoryRoot(state);
-
   return sources
     .valueSeq()
     .toJS()
-    .filter(({ url }) => url && url.includes(root))
+    .filter(({ url }) => url && url.startsWith(root))
```
Comment (darkwing) [inline src/selectors/getRelativeSources.js]: This change, from `includes` to `startsWith`, is causing no sources to display when a root is set.  Do we need this?

Comment (jasonLaster) [reply src/selectors/getRelativeSources.js]: i think startsWith should now work because of the other changes

**Comment by jasonLaster** [comment]
oh nice catch

**Comment by jasonLaster** [comment]
```
Waiting on sources: nested-source
   Finished waiting on sources: nested-source
   Waiting for state change: selected source
   Finished waiting for state change: selected source
 TEST-UNEXPECTED-FAIL Nodes in path are automatically expanded 
   Adding a new tab with URL: http://example.com/browser/devtools/client/debugger/new/test/mochitest/examples/doc-sources.html
  JS warning: 
   "Loading failed for the <script> with source “http://example.com/browser/devtools/client/debugger/new/test/mochitest/examples/nested/deeper/deeper-source.js”." {file: "http://example.com/browser/devtools/client/debugger/new/test/mochitest/examples/doc-sources.html" line: 23}]
   
   ```
   
   oh no, looks like this broke dbg-sources

**Comment by darkwing** [comment]
My local disagrees!  Kicking it.

**Pull request merged**